src/core/main.c

```diff
@@ -11,6 +11,12 @@
 #include <lvgl/lvgl.h>
 #include <minIni.h>
 
+#ifdef EMULATOR_BUILD
+#include "SDLaccess.h"
+#include <SDL2/SDL.h>
```

Comment:
Same issue here.  No need to include SDL.h here either.

Reply:
done